boilerplate/test/test_base_convention/.codecov.yml

```diff
@@ -0,0 +1,26 @@
+codecov:
```

Comment:
Still don't want the files in test conventions to be real files.

Reply:
Even if they don't have to be real life ones, is there any specific issue you are thinking of for having to differ ?

Reply:
Sorry, I'm not quite parsing your question.

I'm just complaining that it's confusing to see real content in these files. For example, somebody in the future is going to think they need to keep them in sync with what's in the real conventions.

It's really not a big deal though.